docs/src/app/sitemap.xml/route.ts

```diff
@@ -0,0 +1,32 @@
+import { apiSource, source } from '../../../lib/source';
+
+export const GET = async () => {
+  const rootUrl = "https://docs.stack-auth.com";
+
+  // Get pages from both sources
+  const docsPages = source.getPages();
+  const apiPages = apiSource.getPages();
+  const allPages = [...docsPages, ...apiPages];
+
+  const sitemap = `<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+${allPages
+  .map((page) => {
+    const url = new URL(page.url, rootUrl);
+    return `  <url>
+    <loc>${url.toString()}</loc>
+    <lastmod>${new Date().toISOString()}</lastmod>
```

Comment:
we don't have this yet, so let's keep it empty as otherwise it may make things worse
```suggestion
```